pkg/skaffold/runner/runcontext/context.go

```diff
@@ -185,6 +185,7 @@ func (rc *RunContext) Tail() bool                                { return rc.Opt
 func (rc *RunContext) Trigger() string                           { return rc.Opts.Trigger }
 func (rc *RunContext) WaitForDeletions() config.WaitForDeletions { return rc.Opts.WaitForDeletions }
 func (rc *RunContext) WatchPollInterval() int                    { return rc.Opts.WatchPollInterval }
+func (rc *RunContext) BuildConcurrency() int                     { return rc.Opts.BuildConcurrency }
```

Comment:
Would I need to call this from builder mux https://github.com/GoogleContainerTools/skaffold/blob/cc262b48867959858e592f76674f8ea0f37e0d8e/pkg/skaffold/build/builder_mux.go#L34

Reply:
In `builder_mux.go` you can add this method to the `Config` interface:
```go

type Config interface {
	GetPipelines() []latest.Pipeline
	DefaultRepo() *string
	GlobalConfig() string
	BuildConcurrency() int  
}
```

In `NewBuilderMux` function, skip the `minConcurrency` [calculations](https://github.com/GoogleContainerTools/skaffold/blob/e947844e558b9181911906399cb5e3df7fdb28ef/pkg/skaffold/build/builder_mux.go#L62-L73) if the value from that method is non-negative (see above comment also).